features/view_specs/view_spec.feature

```diff
@@ -212,3 +212,24 @@ Feature: view spec
       """
     When I run `rspec spec/views`
     Then the examples should all pass
+
+  Scenario: path helpers should take precedence over asset helper
```

Comment:
> also is this a breaking change in our behaviour? Some people may be relying on inverse behaviour already...

I doubt it. This breaks the view test completely, as when you call `video_path` in the _view_ you get `/videos/42` (i.e. it correctly calls routes helper). This collision only occur in the view spec.